go/pkg/cas/upload.go

```diff
@@ -624,40 +624,46 @@ func (u *uploader) check(ctx context.Context, items []*uploadItem) error {
 }
 
 func (u *uploader) scheduleUpload(ctx context.Context, item *uploadItem) error {
+	reqSize := int(marshalledRequestSize(item.Digest))
```

Comment:
nit: while not realistic that we'd ever have more than 2^31-1 items, it's technically safer to do `int` -> `int64` than `int64` -> `int`

Reply:
done